Moya.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
   s.subspec "ReactiveCocoa" do |ss|
     ss.source_files = "Moya/ReactiveCocoa/*.swift"
     ss.dependency "Moya/ReactiveCore"
-    ss.dependency "ReactiveCocoa", "~> 4.0-alpha.1"
+    ss.dependency "ReactiveCocoa", "= 4.0.0-alpha-3"
```

Comment:
Unfortunately theres significant breaking changes in alpha 2 and 3 (which this pull request has changes/fixes for.. you just have to scroll a lot).
So, the point of this pull request is to support the latest alpha, with the expectation that this would be pushed as a 4.1.0 release.